wadas/ui/configure_actuators_dialog.py

```diff
@@ -318,16 +322,22 @@ def accept_and_close(self):
                 if cur_actuator_id and cur_actuator_type:
                     actuators_id.add(cur_actuator_id)
                     if cur_actuator_id in Actuator.actuators:
-                        # actuator exists and we update its enablement status
+                        # Actuator exists and we update its enablement status
                         Actuator.actuators[cur_actuator_id].enabled = cur_actuator_enablement
                     else:
-                        # actuator does not exist, therefore we add it to the dictionary
+                        # Actuator does not exist, therefore we add it to the dictionary
                         if cur_actuator_type == Actuator.ActuatorTypes.ROADSIGN.value:
                             actuator = RoadSignActuator(cur_actuator_id, cur_actuator_enablement)
                             Actuator.actuators[actuator.id] = actuator
+                            # If db is enabled,add actuator into db
+                            if (db:=DataBase.get_instance()) and db.enabled:
+                                DataBase.insert_into_db(actuator)
                         elif cur_actuator_type == Actuator.ActuatorTypes.FEEDER.value:
                             actuator = FeederActuator(cur_actuator_id, cur_actuator_enablement)
                             Actuator.actuators[actuator.id] = actuator
+                            # If db is enabled,add actuator into db
+                            if (db:=DataBase.get_instance()) and db.enabled:
+                                DataBase.insert_into_db(actuator)
             # If an actuator changes the id we have to remove previous orphaned ids from dictionary
             for key in list(Actuator.actuators.keys()):
```

Comment:
.keys() is redundant (dictionaties naturally iterate over their keys):

for key in list(Actuator.actuators):